perception/autoware_tensorrt_bevdet/include/autoware/tensorrt_bevdet/postprocess.hpp

```diff
@@ -0,0 +1,72 @@
+// Copyright 2024 AutoCore, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#ifndef AUTOWARE__TENSORRT_BEVDET__POSTPROCESS_HPP_
+#define AUTOWARE__TENSORRT_BEVDET__POSTPROCESS_HPP_
+
+#include "common.hpp"
+#include "iou3d_nms.hpp"
+
+#include <memory>
+#include <vector>
+
+class PostprocessGPU
+{
+public:
+  PostprocessGPU() {}
+  PostprocessGPU(
+    const int _class_num, const float _score_thresh, const float _nms_thresh,
+    const int _nms_pre_maxnum, const int _nms_post_maxnum, const int _down_sample,
+    const int _output_h, const int _output_w, const float _x_step, const float _y_step,
+    const float _x_start, const float _y_start, const std::vector<int> & _class_num_pre_task,
+    const std::vector<float> & _nms_rescale_factor);
+
+  void DoPostprocess(void ** const bev_buffer, std::vector<Box> & out_detections);
+  ~PostprocessGPU();
+
+private:
+  int class_num;
```

Comment:
Add comments to explain the purpose of each member variable.  For example, for `class_num`, you could add `// Number of object classes detected.`  This improves readability and maintainability.